removeS3ManagedEncryption/function.template.js

```diff
@@ -0,0 +1,22 @@
+var lambdaCfn = require('@mapbox/lambda-cfn');
```

Comment:
Use let instead of var.

Reply:
@matiskay Why `let` instead of `const` ? `let` indicates that the variable is mutable, and IMO the less mutable state a system has, the better. Particularly, one wouldn't want to re-assign a variable representing an imported module. 

(...nevermind that you can still change out properties of `const` objects, rant for another time)

Reply:
@agius @matiskay good catch! This is code I copied from an older Patrol rule using ES5 syntax. I'll switch this to `const`. I'll also look for any other `var`s too and remove.

Reply:
@agius my fault. `const` is better here.